### PR TITLE
[1408] Sort links displayed only if matches are returned

### PR DESCRIPTION
**Pull request by stellenberger**
### Context

- [This trello ticket](https://trello.com/c/vClmil5T/1408-s-only-show-sort-options-if-there-are-results-to-sort)
- The sort links to sort by name or date updated were shown even if there were no results to sort

### Changes proposed in this pull request

- Only display the sort by links if there are sortable results

### Guidance to review

- Start your rails server, navigate to `https://localhost.com` and login using persona or DSI
- Use the search feature, but ensure that no trainees are returned. You may need to delete some in order to achieve this. 
- The sort by links will not be shown when there are no trainees returned in the filter.

### CODE REVIEW

**Comment by stellenberger** [comment]
[[S] Only show sort options if there are results to sort](https://trello.com/c/vClmil5T/1408-s-only-show-sort-options-if-there-are-results-to-sort)

File: spec/features/trainees/filtering_trainees_spec.rb [hunk 1]
```diff
@@ -33,6 +33,12 @@
     then_all_trainees_are_visible
   end
 
+  scenario "no matches" do
+    when_i_filter_by_a_subject_which_returns_no_matches
+    then_i_see_a_no_records_found_message
+    then_i_cant_see_sort_links
```
Comment (darokel) [inline spec/features/trainees/filtering_trainees_spec.rb:39]:
```suggestion
  then_i_should_not_see_sort_links
```

**Review by darokel — APPROVED** [review]

File: app/components/paginated_filter/view.html.erb [hunk 2]
```diff
@@ -6,8 +6,9 @@
     <div class="moj-action-bar">
       <div class="moj-action-bar__filter"></div>
     </div>
-
-    <%= render Trainees::SortLinks::View.new %>
+    <% if !@collection.empty? %>
```
[the remark this answers is not in the document]
Comment (stellenberger) [reply app/components/paginated_filter/view.html.erb:9]: Done 👍

**Comment by GeoffJohnson** [comment]
All good

**Pull request merged**